ansible/roles/flannel/handlers/main.yml

```diff
@@ -5,7 +5,6 @@
     - stop docker
     - delete docker0
     - start docker
-  when: inventory_hostname in groups['nodes']
```

Comment:
docker is no necessarily install on non-nodes, so this is not acceptable as I recall...